components/items/ItemsGrid/ItemsGridImage.vue

```diff
@@ -1,61 +1,30 @@
 <template>
-  <div>
-    <NeoNftCard
-      v-if="item"
-      class="is-hidden-mobile"
-      :nft="item"
-      :prefix="urlPrefix"
-      :show-price="Number(item?.price) > 0" />
-    <NeoNftCard
-      v-if="item"
-      class="is-hidden-tablet"
-      :nft="item"
-      :prefix="urlPrefix"
-      :show-price="Number(item?.price) > 0"
-      variant="minimal" />
-  </div>
+  <NeoNftCard
+    v-if="item"
+    class="is-hidden-mobile"
```

Comment:
render nothing on mobile devices?

![image](https://user-images.githubusercontent.com/22791238/223638301-bebd920e-eb85-45e2-8cf1-16d342958783.png)

Reply:
I agree, i think we are over using this approach and a util/composable for easy access to screen dimensions
and change props accordingly would be better for some cases

Reply:
and btw... change the name of ```isMobileDevice``` in ```@/utils/extension``` to something like ```isTouchScreen```